### ISSUE TITLE
[Bug]: Inconsistent and Sporadic Sync Uploading & Possible Process Failure on Linux Mint 21.3 Cinnamon

### ISSUE THREAD

**Issue by JartanFTW** [nextcloud/desktop#6424]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

When NextCloud Desktop is syncing a local folder to server, upload takes substantially more time while the taskbar menu is not focused.

### Steps to reproduce

1. Have your NextCloud undergo a large syncing process uploading locally to the server. My use-case has many pictures and videos, so this could be related to quantity of files but I doubt it.

3. Open Resource Monitor and observe the Network section for a base-line.
![image](https://github.com/nextcloud/desktop/assets/66337180/ca244611-e4e8-4214-b207-4c399b0e71d4)

4. Click on the NextCloud icon on the taskbar, opening the NextCloud Desktop GUI. Wait for the "Syncing" test to update with an ETA. 
Before update:
![image](https://github.com/nextcloud/desktop/assets/66337180/e80a34ec-df6e-4a36-8766-8ef1f64aa61a)
After update:
![image](https://github.com/nextcloud/desktop/assets/66337180/6e7e38a6-12da-4dde-bc77-25ec18fe4b7d)

5. Once an ETA is presented, observe the Resource Monitor and see how the upload traffic dramatically spikes.
![image](https://github.com/nextcloud/desktop/assets/66337180/8b971f4d-f1c1-46f8-b9d4-3e3dd3f591f9)

7. Defocusing from the NextCloud Desktop window by clicking anywhere else closes the window, dropping upload traffic once more. If you were to open the NextCloud Desktop window once more, you would need to wait for the ETA to be re-generated, and the upload traffic will spike once more.
![image](https://github.com/nextcloud/desktop/assets/66337180/5010a05d-d3f2-4ea7-9384-5f3fc473fe73)

### Expected behavior

NextCloud Desktop should actively sync files without being focused.

### Which files are affected by this bug

N/A

### Operating system

Linux

### Which version of the operating system you are running.

Linux Mint 21.3 Cinnamon

### Package

Distro package manager

### Nextcloud Server version

28.0.2

### Nextcloud Desktop Client version

3.11.0

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
I'm nearly certain that this is a client-side issue based on the behavior, so I'm going to omit this. If this is desired, tag me and I'll look into providing it.

Similarly, I am also not including the Desktop Client Logs as I'm working with sensitive user-data and I found mentions of it immediately when looking at the logs.
```

### Additional info

I have installed NextCloud Desktop on Linux Mint 21.3 Cinnamon via the Software Manager app installing the Flatpak version of NextCloud Desktop, which is currently at 3.11.0 (latest as of writing is 3.11.1).

**Comment by JartanFTW** [comment]
Ok, this might not be as pointed as I originally framed it as I am now experiencing upload speed dropping indefinitely even while I'm still focused on NextCloud Desktop and the ETA Is still visible.
![image](https://github.com/nextcloud/desktop/assets/66337180/2d9184e0-ea29-4487-86a7-fc9043d15af3)

**Comment by JartanFTW** [comment]
After reading through the client logs, I do think providing desktop client logs is likely necessary for this, as I'm seeing some weird errors going on. I'm willing to do this privately with a developer, since I'm working with user-data and don't want that publicized.

**Comment by joshtrichards** [comment]
We had an issue where the tray being *open* would slow things down (#4918) and that was addressed recently in #6691.

> Another behavior I am observing is that if I set the bandwidth limit setting to automatic, the program soon stops responding entirely, after a couple second of normal (high) uploading.

Are you running it with *No limit* otherwise or set to a particular limit?

**Comment by JartanFTW** [comment]
> We had an issue where the tray being _open_ would slow things down (#4918) and that was addressed recently in #6691. That fix is in all recent 3.12.x / 3.13.x releases.
> 
> > Another behavior I am observing is that if I set the bandwidth limit setting to automatic, the program soon stops responding entirely, after a couple second of normal (high) uploading.
> 
> Are you running it with _No limit_ otherwise or set to a particular limit?

At the time I was running it with _No limit_. The problems in this report have long since gone away by now, or gone away enough that I don't notice it in my day to day. I have retained the debug zip folder, though.

**Comment by joshtrichards** [comment]
Sounds like we can close this out in that case. :)
